independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/catalog/CatalogMapperHelper.java

```diff
@@ -19,11 +20,20 @@
 public class CatalogMapperHelper {
 
     private static ObjectMapper mapper;
+    private static ObjectMapper yamlMapper;
```

Comment:
I wondered about this. The yaml/json path was present on the Registry side, but not the catalog side..